integration/tests/cypress_e2e.sh

```diff
@@ -25,29 +25,14 @@ do_deploy() {
                 return 1
             fi
             ;;
-        v2)
+        v2.1)
```

Comment:
you'll want to drop v2 from the [deploy pipeline script](https://github.com/chef/automate/blob/master/.expeditor/buildkite/cypress.sh#L9) too

Reply:
Good catch!